README.md

```diff
@@ -1,6 +1,18 @@
-Amazon Web Services Pricing Ruby Gem
+DEPRECATED: Amazon Web Services Pricing Ruby Gem
 ====================================
 
+⛔️ NOTICE OF FUTURE UPDATES TO THE GEM ⛔️
+====================================
+This project is no longer being maintained and will be deprecated in the future. Issues and pull requests will no longer be accepted. If you have scripts that utilize this project, consider using alternate libraries.
+
+Amazon has updated the method it uses to provide pricing information to be an API; as such the methods contained within this Gem are no longer useful for obtaining pricing information.
+
+Updates to the gem will be few and far between; please contact *nbeavers@vmware.com* with any questions.
```

Comment:
shouldn't direct this to a general mailbox, but specific to pricing, i.e. not a personal address?